src/components/select/ui/ValueContainer/useModifyChildren.tsx

```diff
@@ -0,0 +1,34 @@
+import { Children } from "react";
+import { TAG_SIZE, Tag } from "../../../tag";
+import { SelectProps } from "../../types";
+import { getTagKind, tagOverrides } from "../../sharedUtils";
+
+export const useModifyChildren = (value: SelectProps["value"], isOverflow: boolean, props: any) => {
```

Comment:
It's not a hook but prefix `use` has hookish naming. It's pure function here. It should be named in some other type.

Reply:
it's optional

Reply:
Done.